<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCourseDialog</class>
 <widget class="QDialog" name="AddCourseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>444</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add New Course</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f5f7fa;</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="headerFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #f093fb, stop:1 #f5576c); border-radius: 0px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout" name="headerLayout">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="labelTitle">
        <property name="styleSheet">
         <string notr="true">font-size: 24px; font-weight: bold; color: white; background: transparent;</string>
        </property>
        <property name="text">
         <string>ðŸ“š  Add New Course</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="headerSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonClose">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: rgba(255,255,255,0.2); color: white; border: none; border-radius: 20px; font-size: 18px; font-weight: bold; } QPushButton:hover { background-color: rgba(255,255,255,0.3); }</string>
        </property>
        <property name="text">
         <string>âœ•</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">border: none; background-color: #f5f7fa;</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>505</width>
        <height>284</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f5f7fa;</string>
      </property>
      <layout class="QVBoxLayout" name="contentLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item>
        <layout class="QGridLayout" name="formGrid">
         <property name="horizontalSpacing">
          <number>20</number>
         </property>
         <property name="verticalSpacing">
          <number>15</number>
         </property>
         <item row="10" column="2">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="lineEditCourseCode">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border: 2px solid #E0E0E0;
    border-radius: 10px;
    padding: 8px 10px;
    background: rgba(250, 250, 250, 200); /* slightly transparent soft white */
    font-size: 11pt;
    color: #111111;                         /* user text color */
    font-style: normal;                      /* ensures user text is not italic */
}

QLineEdit:focus {
    border: 2px solid #667EEA;
    background: rgba(250, 250, 250, 220);   /* slightly brighter on focus */
}

QLineEdit::placeholder {
    color: rgba(50, 50, 50, 200);           /* placeholder text color */
    font-style: italic;                      /* placeholder remains italic */
}
</string>
           </property>
           <property name="placeholderText">
            <string>e.g., ECE 101</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lineEditCourseName">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border: 2px solid #E0E0E0;
    border-radius: 10px;
    padding: 8px 10px;
    background: rgba(250, 250, 250, 200); /* slightly transparent soft white */
    font-size: 11pt;
    color: #111111;                         /* user text color */
    font-style: normal;                      /* ensures user text is not italic */
}

QLineEdit:focus {
    border: 2px solid #667EEA;
    background: rgba(250, 250, 250, 220);   /* slightly brighter on focus */
}

QLineEdit::placeholder {
    color: rgba(50, 50, 50, 200);           /* placeholder text color */
    font-style: italic;                      /* placeholder remains italic */
}
</string>
           </property>
           <property name="placeholderText">
            <string>e.g., Digital Logic Design</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="labelCourseName">
           <property name="styleSheet">
            <string notr="true">font-size: 13px; color: #718096; font-weight: bold; background: transparent;</string>
           </property>
           <property name="text">
            <string>Course Name *</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelCourseCode">
           <property name="styleSheet">
            <string notr="true">font-size: 13px; color: #718096; font-weight: bold; background: transparent;</string>
           </property>
           <property name="text">
            <string>Course Code *</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelCreditHours">
           <property name="styleSheet">
            <string notr="true">font-size: 13px; color: #718096; font-weight: bold; background: transparent;</string>
           </property>
           <property name="text">
            <string>Credit Hours *</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLineEdit" name="lineEditCreditHours">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border: 2px solid #E0E0E0;
    border-radius: 10px;
    padding: 8px 10px;
    background: rgba(250, 250, 250, 200); /* slightly transparent soft white */
    font-size: 11pt;
    color: #111111;                         /* user text color */
    font-style: normal;                      /* ensures user text is not italic */
}

QLineEdit:focus {
    border: 2px solid #667EEA;
    background: rgba(250, 250, 250, 220);   /* slightly brighter on focus */
}

QLineEdit::placeholder {
    color: rgba(50, 50, 50, 200);           /* placeholder text color */
    font-style: italic;                      /* placeholder remains italic */
}
</string>
           </property>
           <property name="placeholderText">
            <string>e.g., Dr. John Smith</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="footerFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white; border-top: 2px solid #e0e0e0;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout" name="footerLayout">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <spacer name="footerSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCancel">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>45</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #f8f9fa;
    color: #5a6c7d;
    border: 2px solid #e0e0e0;
    border-radius: 8px;
    font-size: 14px;
    font-weight: bold;
    padding: 10px 20px;
}

QPushButton:hover {
    border-color: #f5576c;
    color: #f5576c;
}

QPushButton:pressed {
    background-color: #e0e0e0; /* darker grey for pressed feedback */
    border-color: #f5576c;
    color: #f5576c;
}</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSave">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>45</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #f5576c;
    color: #ffffff;
    border: none;
    border-radius: 8px;
    font-size: 14px;
    font-weight: bold;
    padding: 10px 20px;
    transition: background-color 150ms;
}
QPushButton:hover {
    background-color: #e5475c;
}
QPushButton:pressed {
    background-color: #d43f52;
}
</string>
        </property>
        <property name="text">
         <string>ðŸ’¾  Save Course</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
